frontend/src/containers/Subscription/SubscriptionFeed.js

```diff
@@ -20,6 +59,77 @@ class SubscriptionFeed extends Component {
     //     }
     // }
 
+    clickMoreButton = () => {
+        if (!this.props.subscriptionFinished) {
+            this.props.onGetSubscriptions({
+                page_number: this.props.subscriptionPageNum + 1,
+            })
+                .then(() => {
+                    const { subscriptions } = this.state;
+                    this.setState({
+                        subscriptions: subscriptions.concat(this.props.subscriptionItems),
+                    });
+                    this.clickMoreButtonNext();
+                }).catch(() => {});
+        } else {
+            this.clickMoreButtonNext();
+        }
+    }
+
+    clickMoreButtonNext = () => {
+        if (!this.props.recommendationFinished) {
+            this.props.onGetRecommendations({
+                page_number: this.props.recommendationPageNum + 1,
+            })
+                .then(() => {
+                    const { recommendations } = this.state;
+                    this.setState({
+                        recommendations: recommendations.concat(this.props.recommendationItems),
+                    });
+                    this.addRecoToSub();
+                }).catch(() => {});
+        } else {
+            this.addRecoToSub();
+        }
+    }
+
+    addRecoToSub = () => {
```

Comment:
[Just for Check] Did you check after clicking 'View More' button, cards which have been before are not rearranged?

Reply:
Yes!